### ISSUE TITLE
Crash when starting up

### ISSUE THREAD

**Issue by Wootter** [OpenShot/openshot-qt#5165]
**Describe the bug**
A clear and concise description of the issue

**Steps to reproduce the behavior:**
1. Click on OpenShot to start it up
2. Wait on the application to load
3. Click a phew time and wait 5ish second after
4. See app freeze
5. See app crash

**Expected behavior**

**System Details**
- OpenShot Version: 3.0.0
- libopenshot Version:  0.3.0
- Platform: Windows-10-10.0.19041
- Processor: Unknown
- Machine: AMD64
- Python version: 3.8.9
- Qt5 version: 5.15.2
- PyQt5 version: 5.15.4
- Qt Detected Languages: ['nl-NL', 'en-US']
- LANG Environment Variable:  School
- LOCALE Environment Variable:  /
- Daily Build: Skipped this step

**Log Files**
- [openshot-qt.log](https://s3.amazonaws.com/logs.openshot.org/1tg1sqaulytn42y49cr7lsghaoxoz455/openshot-qt.zip) (4 KB)
- libopenshot.log: File too small or empty (just the start-up message)

**Exception / Stacktrace**
```No stacktrace found in log files```

**Screenshots** (Optional)
If applicable, add screenshots to help explain your problem. You can include screenshots by
copy/pasting them on GitHub or dragging-and-dropping into the GitHub page. All images are public,
so please don't post screenshots containing personal information.

**Comment by Wootter** [comment]
I need this app for a school assignment to so faster the better

**Comment by Colorjet3** [comment]
Hello @Wootter.  The production release of v3.0.0-Release-Candidate has been buggy on some Windows 10 and 11 builds.   A fix has been release in the latest DEV Daily Build.  Please do this:

1.  Go to Add or Remove Programs and uninstall Openshot.
2.  Backup your c:\users\username\.openshot_qt folder.
3.  Delete c:\users\username\.openshot_qt folder.
4.  go to openshot.org/download/#daily and download the latest DEV Daily build available.
5.  Install it.
6.  Start Openshot and go to Edit | Preferences | Preview tab.  Make sure that the Default Audio Sample Rate is set to 48000.  If you had to change it to this, then Restart Openshot.

You should be set to go.  Please confirm.

**Comment by rainieryoung** [comment]
I'm experiencing a similar issue on MacOS. Both the 3.0 build and the latest daily build.

While working on a project, Openshot crashed.
1. I can open the openshot program again, it will default to an empty project
2. When I open the project that I was working on, it starts loading files (I see the timeline, project files, and video preview), but crashes again within seconds

At the time of the crash, I do see an error in **libopenshot.log**

```
------------------------------------------
libopenshot logging: Fri Mar 10 13:47:08 2023
------------------------------------------
---- Unhandled Exception: Stack Trace ----
---- End of Stack Trace ----
```

In **openshot-qt.log**, there is no record of an error or crash until I re-open Openshot. At that time, I will see...

```
13:48:49 ERROR main_window: Unhandled crash detected: No Log Detected
```

**Comment by Colorjet3** [comment]
Hello @rainieryoung.  I don't know much about the Mac environment and don't have one to play with.  If you don't mind please upload your entire libopenshot.log file.  I am going to ask one of the developers to take a look at this.

**Comment by rainieryoung** [comment]
Attached: [libopenshot.log](https://github.com/OpenShot/openshot-qt/files/10948870/libopenshot.log)

Apologies for the delay, I didn't see a notification when you replied. I'll look into my github notification settings and take care of that.

**Comment by Colorjet3** [comment]
This has been fixed in v3.1.1
